### ISSUE TITLE
helm update rancher from v2.3.6 to v2.5.5 ， update faild

### ISSUE THREAD

**Issue by alec-lu** [rancher/rancher#31472]
*Describe your issue here*

Error: UPGRADE FAILED: rendered manifests contain a resource that already exists. Unable to continue with update: Ingress "rancher" in namespace "cattle-system" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "rancher"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "cattle-system"
---
| Useful | Info |
| :-- | :-- |
|Versions|Rancher `v2.3.6` UI: `v2.3.40` |
|Route|`global-admin.clusters.index`|

**Comment by alec-lu** [comment]
分为2个步骤升级，已经解决
helm get values rancher -n cattle-system -o yaml > values.yaml 
helm upgrade rancher rancher-latest/rancher \
  --namespace cattle-system \
  -f values.yaml \
  --version=2.4.12
helm upgrade rancher rancher-latest/rancher \
  --namespace cattle-system \
  -f values.yaml \
  --version=2.5.5

**Comment by throker** [comment]
BTW: Google Translate says @alec-lu Chinese sentence in English is " The upgrade is divided into 2 steps and has been resolved"

I had the same exact issue, I was trying to upgrade to the latest stable so I first upgraded to 2.4.12 like above. 
Let it install and waited for the cluster to settle down.
Then for the second i left the specified version off so it would install the latest stable.

This issue can be closed, but a note or something should be added to the Docs for those that are needing to upgrade from rancher 2.3.6 to anything over 2.4.12.
